composer.json

```diff
@@ -34,5 +34,10 @@
       "dealerdirect/phpcodesniffer-composer-installer": true,
       "composer/installers": true
     }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "PerformanceLab\\Tests\\": "tests/utils"
+    }
   }
```

Comment:
Not sure Composer usage like this would play well together with WordPress core, but since it's only tests IMO it should be okay at least in terms of this plugin. The classes would need to be renamed for core anyway.

Reply:
Do you mean I need to rename util classes to follow `class-...` pattern? If so, can we make an exception for phpunit classes only?

Reply:
WordPress core still doesn't use _any_ autoloading, there's always some sort of "includes" file that `require_once`s all files. Obviously not great, but that's how I would do it, for maximum "compatibility" with the core paradigms.

However, since this is purely about PHPUnit classes and we're in this plugin, I'm not opposed to this (obviously better) approach :)

Reply:
Awesome! Thanks 😄